gef.py

```diff
@@ -3766,12 +3766,25 @@ def dereference(addr: int) -> Optional["gdb.Value"]:
     return None
 
 
-def gef_convenience(value: str) -> str:
-    """Defines a new convenience value."""
+def gef_gen_convenience_name() -> str:
```

Comment:
Probably one final thing here
- either make the name explicit `gef_generate_convenience_name()`
- or keep it as-is and make this function internal to `gef_convenience` since it's not used anywhere else (and unlikely to be).

Reply:
done!